scripts/add_new_provider.py

```diff
@@ -181,12 +205,9 @@ def get_provider_info(provider_info={}):
         # 3.  Collect provider type
         print "Select a provider type for your new provider"
         print "1: Openstack"
-        while True:
-            provider_type = raw_input("Select a provider type [1]: ")
-            #NOTE: this will be replaced with actual logic when necessary.
-            if True:
-                provider_type = openstack
-                break
+        provider_type = require_input("Select a provider type [1]", default='1')
```

Comment:
Should this be `provider_type = require_input("Select a provider type [1]", default=openstack)`?